docs/calendar-review.md

```diff
@@ -0,0 +1,212 @@
+
+### Alternative 2: Separate ISO and Calendar Properties
+
+It would be possible to retain the calendar in Temporal objects’ data model without being vulnerable to the Unexpected Calendar Problem, but doing so would involve a significant increase in Temporal surface area. Below is one way it could be done. 
+
+*   Offer two sets of fields, e.g. `.isoMonth` vs. `.calendarMonth` (or `month` vs. `calendarMonth` for brevity and to encourage developers who are unfamiliar with calendars to be nudged to use ISO fields?)
+    *   The number and types of `calendar*` fields are completely up to the calendar implementation.  Built-in calendars will all offer `calendarDay`, `calendarMonth`, `calendarMonthCode`, `calendarYear`, and sometimes `calendarEra` and `calendarEraYear` too.
+    *   Temporal APIs that accept property bag parameters will accept either calendar fields or ISO fields, BUT NOT BOTH. `date.with({calendarYear: 5780, isoMonth: 12}) `should throw. BTW, this may be enforceable via TS and ESLint for a nice design-time DX.
+    *   Trying to read `calendar*` fields would throw if the calendar is ISO because there is no human calendar defined.
+    *   Non-field convenience properties (e.g. `dayOfYear`, `inLeapYear`, etc.) would also be duplicated. The non-`calendar` variants of these fields would report the ISO values.
+*   `Temporal.Duration` would gain a `calendar` field.  
+    *   Instances using the ISO calendar would only support ISO fields: `days`, `weeks`, `months`, and `years`. 
+    *   Instances using a non-ISO calendar would only support calendar fields: `calendarDays`, `calendarWeeks`, `calendarMonths`, and `calendarYears`. 
+    *   String parsing and serialization would use calendar annotations like other Temporal types, e.g. `'P2D[u-ca-chinese]'`. Strings without an annotation would assume the ISO calendar.
+    *   Question: should `date.add({calendarMonths: 1})` use `date.calendar`? Or should `date.add({calendarMonths: 1, calendar: 'chinese'})` be required even if `date.calendar==='chinese'`?
+    *   `until` and `since` are now ambiguous, so they’d need a way to indicate whether the output should be an ISO duration or a non-ISO one. Options include:
+        *   Split each method into "iso" and "calendar" variants, e.g. `isoUntil` vs. `calendarUntil` or `untilISO` vs. `untilCalendar`
+        *   Always return both flavors, e.g. `foo.until(bar).isoDuration` and `foo.until(bar).calendarDuration`.
+        *   Use an option, e.g. `{ resultFields: 'iso' | 'calendar' }`. FWIW, I'm not sure that this option would work well with TS in cases where options are not specified as a literal in the API call.
+    *   There would NOT be a `withCalendar` method on durations, because durations don’t have the ability to convert between calendar and ISO data. Instead, they would either have ISO fields or calendar fields, but not both.
```

Comment:
```suggestion
    *   There would NOT be a `withCalendar` method on durations, because durations don't have the ability to convert between calendar and ISO data. Instead, they would either have ISO fields or calendar fields, but not both.
```